code/modules/admin/verbs/adminhelp.dm

```diff
@@ -140,7 +140,7 @@
 		if(!adm["afk"] && !adm["stealth"] && !adm["noflags"])
 			send2irc(source, "[msg] - No admins online")
 		else
-			send2irc(source, "[msg] - All admins AFK ([adm["afk"]]/[adm["total"]]), stealthminned ([adm["stealth"]]/[adm["total"]]), or lack [requiredflags] ([adm["noflags"]]/[adm["total"]])")
+			send2irc(source, "[msg] - All admins AFK ([adm["afk"]]/[adm["total"]]), stealthminned ([adm["stealth"]]/[adm["total"]]), or have no power ([adm["noflags"]]/[adm["total"]])")

```

Comment:
Nitpicky, but this should probably be explicit in that they lack ban flags rather than just 'have no power'
Doesn't really matter however